tests/common/jwt/jwt_token_test.go

```diff
@@ -32,12 +32,13 @@ func getKey(jwksFile string, t *testing.T) interface{} {
 	if err != nil {
 		t.Fatalf("failed to read jwks: %s", err)
 	}
-	jwks, err := jwk.ParseBytes(data)
+	jwks, err := jwk.Parse(data)
```

Comment:
I am not concerned at all about updating this security-sensitive library since it is used exclusively in tests.

Reply:
Ha .. it's only used in tests? Should it be moved to `pkg/test`?